### PR TITLE
Remove yiisoft/yii-core dependency

### PR DESCRIPTION
**Pull request by viktorprogger**
| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | no
| Fixed issues  | #12 

Waiting for fix in yii-core to run tests.

### CODE REVIEW

File: src/Cache.php [hunk 1]
```diff
@@ -7,10 +7,9 @@
 
 namespace Yiisoft\Cache;
 
-use yii\base\Component;
+use yii\helpers\Yii;
```
Comment (samdark) [inline src/Cache.php]: Isn't it part of `yii-core` package?

**Comment by viktorprogger** [comment]
I have no idea for now how to change the migration. Could you provide a suggestion?

**Comment by samdark** [comment]
No. Migrations are undefined for now.

**Comment by viktorprogger** [comment]
BTW, this PR is ready for review :)

**Comment by samdark** [comment]
There's another one https://github.com/yiisoft/cache/pull/15. Will try to take best from both.

**Comment by viktorprogger** [comment]
I suggest to create a feature branch with #15 and I'll make a PR with my changes there.

**Comment by samdark** [comment]
Going to finish/merge #15 first. This one has some good things such as custom exceptions.

**Comment by samdark** [comment]
Applied changes giving credits. Will push soon.

**Pull request closed without merging**